src/compiler/scala/tools/ant/templates/tool-unix.tmpl

```diff
@@ -127,6 +127,11 @@ fi
 
 [[ -n "$JAVA_OPTS" ]] || JAVA_OPTS="@javaflags@"
 
+# enable colored output by default
+# we prepend it to JAVA_OPTS so users can override it on the
+# command line or via JAVA_OPTS
+[[ -t 1 ]] && JAVA_OPTS="-Dscala.color=true $JAVA_OPTS"
```

Comment:
I originally didn't want to clutter the command line to run with one more variable, but you're right, it's cleaner that way. Done.